csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -5572,6 +5574,33 @@ A distributing party satisfies the "CSAF aggregator" role if the party:
 > CSAF aggregators can be provided for free or as a paid service.
 > To aid in automation, CSAF aggregators MAY mirror CSAF documents from CSAF publishers. Regarding the terms of use they SHOULD consult with the issuing party.
 
+## 7.3 Retrieving rules
+
+The retrieving process splits up into two parts: Finding the `provider-metadata.json` (requirement 7 in section 7.1) and retrieving CSAF documents.
+
+> A retrieving party MAY choose to do the first part only when adding new or updating distributing parties. However, it is recommended to check regulary whether new information is available.
+
+### 7.3.1 Finding provider-metadata.json
+
+**Direct locating**: The following process SHOULD be used to determine the location of a `provider-metadata.json` (requirement 7 in section 7.1) based on the main domain of the issuing party:
+
+1. Checking the Well-known URL (requirement 9 in section 7.1)
```

Comment:
The first three steps may profit from replacing the term `checking` With a more specific activity. I know this is tricky, but if we can be more specific, maybe the implementers can comply more easily.

Reply:
Agree, but I'm currently missing a better word.

Reply:
Retrieving, requesting?

Reply:
[4962a82](https://github.com/oasis-tcs/csaf/pull/430/commits/4962a8231345674be8b7d8064d995432d8b098d6): not ideal but at start.